components/Question.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+const Header = () => {
+    return (
+      <div style={{ backgroundColor: 'white', position: 'fixed', top: 0, left: 0, right: 0, zIndex: 9999 }}>
+        <h1 style={{ textAlign: 'center', margin: 0, padding: '20px' }}>Quiz-App</h1>
+      </div>
+    );
+  };
+
+const Question = ({ category, question }) => {
+    return (
+        <div>
+          <Header />
```

Comment:
You are rendering the `<Header />` twice. Once here, and once in your `pages/index.js`. I think you can remove it here.
```suggestion
```

Reply:
If you remove the Header here, you also do not need the surrounding `<div>`.